common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -283,6 +283,18 @@
                 sequenceLinks = this.content_container.find('a.seqnav');
                 sequenceLinks.click(this.goto);
 
+                // Some Xblocks may cause the window to scroll to the bottom of
+                // the page on initialization (probably a bug). The call to
+                // focus() below generally brings the main content back onto
+                // screen, however, this behaviour is inconsistent both within
+                // and between different browsers. 
+                //
+                // To make this consistent we automatically scroll so the unit
+                // content is at the top of the viewport. Note: we do a hard
+                // scroll rather than a smooth animation since this is on page
+                // load and we don't need any visual transition.
+                $('html > body').scrollTo(this.content_container.offset().top, 0);
+
                 this.sr_container.focus();
```

Comment:
@tartansandal  The [eslint quality diff failure](https://build.testeng.edx.org/job/edx-platform-quality-pipeline-pr/14225/Diff_20Quality_20Report/) is complaining about something in this file: 

> common/lib/xmodule/xmodule/js/src/sequence/display.js | 91.7% | 290: Error - Trailing spaces not allowed. (no-trailing-spaces)

Could you please address?

Reply:
Ah must check "Diff Quality Report" in future. Thanks for that :+1: